configure.ac

```diff
@@ -1933,6 +1969,7 @@ echo
 echo "Options used to compile and link:"
 echo "  external signer = $use_external_signer"
 echo "  multiprocess    = $build_multiprocess"
+echo "  with experimental syscall sandbox support (-sandbox=<mode>) = $use_syscall_sandbox"
```

Comment:
This is supposed to be a summary, not documentation :smile: Please shorten this to something like `echo "  with experimental syscall sandbox support"`

Reply:
Makes sense. Fixed!